operations/github/onboarding/onboarding.sh

```diff
@@ -223,11 +231,11 @@ MAIN_TEAM_PARENT=$(echo "$MAIN_TEAM" | jq -r '.parent_team')
 MAIN_TEAM_PRIVACY=$(echo "$MAIN_TEAM" | jq -r '.privacy')
 MAIN_TEAM_NOTIFICATION=$(echo "$MAIN_TEAM" | jq -r '.notification_setting')
 
-# Get parent team ID
+#Get parent team ID
```

Comment:
Weird change. Usually the comment symbol "#" is followed by a space. It doesn't *matter* syntactically, but I fear readers might misunderstand this line.